pandas/tests/window/moments/test_moments_consistency_rolling.py

```diff
@@ -136,6 +136,54 @@ def test_rolling_apply_consistency(
                 tm.assert_equal(rolling_f_result, rolling_apply_f_result)
 
 
+@pytest.mark.slow
```

Comment:
why do you need the _rolling_consistency_cases here? rather than simply constructing a set of cases.

Reply:
That's fair, I'm open to creating an additional test if you'd like me to. If we have an array of numbers `arr` and dfs:
```python
base_df = DataFrame({'data': arr})
group_df = DataFrame({'group': ['A']*len(arr) + ['B']*len(arr), 'data': arr + arr})
```
The idea here was just to test that the result of `base_df.rolling...` and the result of each group in `group_df.groupby('group').rolling...` are all identical. Because in the current master branch, the `base` result is different then the `A` result which is also different from the `B` result.

Reply:
Essentially I'm testing the consistency of using `df.rolling` vs `df.groupby.rolling` (and also testing the consistency of different groups within `df.groupby.rolling`) since that was the issue before.

Reply:
Yeah I think it'd be better to add test cases to `window/test_grouper.py` in with more `pytest` idioms